src/System.Management.Automation/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -987,8 +987,25 @@ internal override void Initialize()
                     columnWidthsHint = tableHint.columnWidths;
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.http://shop.oreilly.com/product/9780596801519.do should work
```powershell
retry of URL failed with error: Response status code does not indicate success: 504 (Gateway Time-out).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```